reachy_sdk/reachy_sdk.py

```diff
@@ -320,6 +320,28 @@ def turn_off_smoothly(self, part, duration: float = 2.0):
         for joint in req_part.joints.values():
             joint.torque_limit = 100.0
 
+    def set_torques(self, part, percent: float = 100.0):
```

Comment:
Do we want the method to be exposed or hidden as
```python
def _set_torques(self, part, percent)
```
?

I find it a bit too much to have this exposed at the same level as _reachy.r_arm_, _reachy.l_arm_,  _reachy.head_ ...

Reply:
Agreed for both remarks. I added a commit that fixes both of them. Thanks for the review.